psi4/driver/driver_cbs.py

```diff
@@ -25,34 +25,152 @@
 #
 # @END LICENSE
 #
+"""Plan, run, and assemble QC tasks to obtain composite method, basis, & options treatments.
+
+===
+CBS
+===
+
+cbs_text_parser()
```

Comment:
This may or may not fire depending on whether the use passed a string or kwargs, right?

Reply:
pretty much. `cbs_text_parser` is called from task_planner and fires whenever `"/"` is in the method string. it transforms into a set of cbs kwargs that may (if one of the kwargs is a cbs dict spec) trigger a subsequent composite wrapper invocation or may not (it might have been a "b3lyp/6-31G").

later on, called from the composite `__init__` is `_process_cbs_kwargs()`. that fires trivially when the user passed a cbs dict spec. or it does real work when the user passed cbs kwargs.

I've added some clarifying bullets.